### ISSUE TITLE
Load balancer counters are reset to 0 if there are load balancer changes

### ISSUE THREAD

**Issue by weizhouapache** [apache/cloudstack#6849]
CloudStack uses haproxy in VR to implement the load balancer feature. If there are any load balancer changes (add/remove VMs, add/remove rules, etc), the haproxy configuration will be regenerated and haproxy service in VR will be reloaded.

We've found an issue that counters are reset to 0 after haproxy reload, which is incorrect because the existing connections are remain alive. 

Before haproxy reload:
![image](https://user-images.githubusercontent.com/57355700/197732623-c1d9bb5e-dc71-41f1-a66d-b9b756319560.png)

After haproxy reload:
![image](https://user-images.githubusercontent.com/57355700/197732986-95fcf89d-afb0-4ff1-aeb0-5a9f1e514010.png)

An issue has been created in haproxy community: https://discourse.haproxy.org/t/stats-are-reset-to-0-after-haproxy-reload/8051

Users who monitor the load balancer connections, please pay attention to it.

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
VR
~~~

##### CLOUDSTACK VERSION

~~~
4.17
4.18
~~~

##### CONFIGURATION

Advanced network
load balancer

##### STEPS TO REPRODUCE

~~~

~~~

##### EXPECTED RESULTS

~~~

~~~

##### ACTUAL RESULTS

~~~

~~~

**Comment by weizhouapache** [comment]
A possible solution for this
https://discourse.haproxy.org/t/stats-are-reset-to-0-after-haproxy-reload/8051/9

However, cloudstack VR 4.17.1.0/4.18.0 uses haproxy 2.2.9, which is too old.

**Comment by rohityadavcloud** [comment]
Does that impact usage records @weizhouapache ? That may be a blocker/critical issue if our usage records are historically incorrect.

**Comment by weizhouapache** [comment]
@rohityadavcloud 
no. Cloudstack does not generate usage records for load balancer connections.
It only impacts autoscaling feature.
